mage_ai/frontend/pages/datasets/[slug].tsx

```diff
@@ -190,14 +244,119 @@ function Data() {
 
   const headEl = (
     <FlexContainer alignItems="justify-right" flexDirection="row-reverse" >
-      <Button 
-        onClick={viewColumns}
-      >
+      <Button onClick={viewColumns}>
         <Text bold> Column view </Text>
       </Button>
     </FlexContainer>
   );
 
+  type ActionProps = {
+    idx: number;
+    link?: () => void;
```

Comment:
The property name `link` seems a bit misleading. Initially thought it was an object versus an action.

Reply:
I'll make notes of all of these comments as P2 updates and make double-checking skipped/applied suggestions as P1.